sktime/datatypes/_adapter/gluonts.py

```diff
@@ -36,15 +39,15 @@ def convert_pandas_to_listDataset(pd_dataframe: pd.DataFrame):
     >>> from sktime.datatypes._adapter.gluonts import convert_pandas_to_listDataset
     >>> X_list_dataset = convert_pandas_to_listDataset(X)
     """
-    from gluonts.dataset.common import ListDataset
-    from gluonts.dataset.field_names import FieldName
+    # use numpy to implement abstraction of gluonTS ListDataset
+    import numpy as np
 
     # For non-multiindexed DataFrames
     if not isinstance(pd_dataframe.index, pd.MultiIndex):
         start_datetime = pd_dataframe.index[0]
 
-        target_columns = pd_dataframe.columns[:]
-        target_values = pd_dataframe[target_columns]
+        target_columns = pd_dataframe.columns.difference(["instances", "timepoints"])
+        target_values = pd_dataframe[target_columns].to_numpy().astype(np.float32)
```

Comment:
I haven't followed the full discussion, so apologies if it's discussed already. Just had a quick question: how are you deciding float32? Does it not support uint/int/etc. other types, and more importantly float64?

Reply:
Well, it's fine then, but I do find this surprising on the shared link. I'd have expected uint for categorical and float64 for numerical.

> If `is_cat=True`, the array type is `np.int32`, otherwise it is `np.float32`.

But if gluonts follows this, then we're all good wrt this mtype PR. 👍🏽